src/contracts/interfaces/IVoteWeigher.sol

```diff
@@ -1,25 +1,95 @@
 // SPDX-License-Identifier: BUSL-1.1
 pragma solidity =0.8.12;
 
+import "../interfaces/IStrategyManager.sol";
+import "../interfaces/IServiceManager.sol";
+import "../interfaces/ISlasher.sol";
+import "../interfaces/IDelegationManager.sol";
+
 /**
  * @title Interface for a `VoteWeigher`-type contract.
  * @author Layr Labs, Inc.
  * @notice Terms of Service: https://docs.eigenlayer.xyz/overview/terms-of-service
  * @notice Note that `NUMBER_OF_QUORUMS` is expected to remain constant, as suggested by its uppercase formatting.
  */
 interface IVoteWeigher {
+    /**
+     * @notice In weighing a particular strategy, the amount of underlying asset for that strategy is
+     * multiplied by its multiplier, then divided by WEIGHTING_DIVISOR
+     */
```

Comment:
should we include the `WEIGHTING_DIVISOR` in this interface too? seems "proper" to me

Reply:
https://github.com/Layr-Labs/eigenlayer-contracts/pull/76/commits/f616be4a4eacaadcc5b99c15c5d296736a8bd284